tools/release

```diff
@@ -45,44 +45,45 @@ print()
 
 # Perform local backup, go to tools dir to run it.
 cd(tooldir)
-sh('./make_tarball.py')
-sh('mv ipython-*.tgz %s' % ipbackupdir)
 
-# Build release files
-sh('./build_release %s' % ipdir)
+if 'upload' in sys.argv:
```

Comment:
This looks like we now have essentially two scripts in one file. Would it be tidier to separate them?